src/main/java/org/ow2/proactive/catalog/rest/controller/BucketController.java

```diff
@@ -135,13 +139,59 @@ public BucketMetadata getMetadata(
                 groups = Collections.singletonList(ownerName);
             }
 
-            return bucketService.listBuckets(groups, kind);
+            return bucketService.listBuckets(groups, kind, contentType);
 
         } else {
-            return bucketService.listBuckets(ownerName, kind);
+            return bucketService.listBuckets(ownerName, kind, contentType);
         }
     }
 
+    /*
+     * @ApiOperation(value = "Lists the buckets with specific content type")
+     * 
+     * @ApiResponses(value = { @ApiResponse(code = 401, message = "User not authenticated"),
+     * 
+     * @ApiResponse(code = 403, message = "Permission denied"), })
+     * 
+     * @RequestMapping(method = GET)
+     * public List<BucketMetadata> listWithContentType(
+     * 
+     * @ApiParam(value = "sessionID", required = false) @RequestHeader(value = "sessionID", required
+     * = false) String sessionId,
+     * 
+     * @ApiParam(value = "The name of the user who owns the Bucket") @RequestParam(value = "owner",
+     * required = false) String ownerName,
+     * 
+     * @ApiParam(value = "The kind of objects that buckets must contain") @RequestParam(value =
+     * "kind", required = false) String kind,
+     * 
+     * @ApiParam(value =
+     * "The content type of objects that buckets must contain") @RequestParam(value = "contentType",
+     * required = false) String contentType)
+     * 
+     * throws NotAuthenticatedException, AccessDeniedException {
+     * if (sessionIdRequired) {
+     * RestApiAccessResponse restApiAccessResponse =
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)